examples/swr-site/pages/remote/graphql-eslint/_meta.en-US.js

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable */
+const { listFiles } = require("../../../components/remote-utils");
+
+module.exports = async () => {
+  const filePaths = (await listFiles()).filter(
+    (filename) =>
+      filename.startsWith("website/src/pages/docs") && /\.mdx?$/.test(filename)
+  );
+  const paths = filePaths.map((filename) =>
+    filename
+      .replace("website/src/pages/docs/", "")
+      .replace(/\.mdx?$/, "")
+      .split("/")
+  );
+
+  const metaMap = Object.create(null);
+
+  for (const path of paths) {
+    const dirPath = "/" + path.slice(0, -1).join("/");
+    const filename = path.at(-1);
+
+    metaMap[dirPath] ||= {};
+    metaMap[dirPath][filename] = filename;
+  }
+
+  // TODO: return meta from metaMap
+  return {
+    "/": {
+      configs: "configs",
+      "custom-rules": "custom-rules",
+      "getting-started": "getting-started",
+      "/getting-started": {
+        "parser-options": "parser-options",
+        parser: "parser",
+      },
```

Comment:
Have you considered the `items` proposal from #1239? Or supporting `/` in keys like:

```json
{
  "configs": "configs",
  "getting-started": "getting-started",
  "getting-started/parser-options": "parser-options",
  "getting-started/parser": "parser"
}
```

Reply:
I think we can support boths?

Reply:
I think they need to be merged right? It's still 1 item on the sidebar 🤔

Reply:
they should not be merged so nextra under the hood will know if folder has index page or not

Reply:
The tricky thing was i18n. If users can choose `foo.en.mdx` or `foo/index.en.mdx` (they point to the same URL), and when accessing `/foo`, the middleware doesn't know if it's `foo.en` or `foo/index.en`. So in the past I documented it as you'll always have to avoid `index.en.mdx`.

But I think overall the middleware based i18n solution is too hacky and I'd like to fix it properly. I have an idea for that and I'll probably start working on it soon!